### PR TITLE
Destroy repo_server containers in test-upgrade.sh

### PR DESCRIPTION
**Pull request by alextricity25**
This commit adds bits to the test-upgrade.sh so the repo_server
containers are destroyed per https://github.com/rcbops/u-suk-dev/issues/199

Connects https://github.com/rcbops/u-suk-dev/issues/199

### CODE REVIEW

**Comment by evrardjp** [comment]
Are we sure we want to use lxc-container-destroy?

This is gonna destroy logs, etc.

Maybe we should have a --skip-tags container-directories ?

**Comment by sigmavirus24** [comment]
@evrardjp if there's a chance leaving logs around, etc. will cause instability/reproducibility problems for support, then it's likely we'll end up destroying them in the end anyway

File: scripts/test-upgrade.sh [hunk 1]
```diff
@@ -29,3 +29,12 @@ RPCD_DIR="$BASE_DIR/rpcd"
 #        to fail to run.
 cd ${RPCD_DIR}/playbooks
 ansible -m file -a "path=/var/www/repo/openstackgit/horizon-extensions state=absent" repo_all
```
Comment (sigmavirus24) [inline scripts/test-upgrade.sh]: We don't need this if we're going to destroy the repo containers, right? Let's remove this now.

**Comment by sigmavirus24** [comment]
This needs a bit more work. 👎

**Comment by alextricity25** [comment]
@evrardjp: this is the same pattern OSA uses in their K->L upgrade script. Nothing new here.
@sigmavirus24 Good point! I'll remove that

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/staging.horse/build-2351c57221/docs/private-cloud/rpc/release-notes/).

**Comment by hughsaunders** [comment]
👍  pending jenkins.

**Comment by evrardjp** [comment]
fine for me if everybody agrees on what it does.

**Comment by sigmavirus24** [comment]
👍 once the checks pass

**Comment by prometheanfire** [comment]
:+1: once passing

**Comment by alextricity25** [comment]
recheck_ceph

**Comment by alextricity25** [comment]
:-1: In the OSA mitaka upgrade script, there is a [playbook](https://github.com/openstack/openstack-ansible/blob/stable/mitaka/scripts/run-upgrade.sh#L153) that targets the repo_server containers, and it runs before the setup-hosts.yml playbook.
I'm going to add ``setup-hosts.yml --limit repo_all`` here to rebuild the containers before said playbook is run.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/staging.horse/build-859ff14f7f/docs/private-cloud/rpc/release-notes/)

**Comment by alextricity25** [comment]
Alright, I made the change. Please review the latest commit :)

**Comment by git-harry** [comment]
👍  this looks good to me. OSA's run-upgrade.sh script does create some potential issues because it performs certain tasks related to the vars and env files however in practise I don't believe they will cause any problems.

**Comment by mattt416** [comment]
LGTM 👍

**Pull request merged**